pkg/chunked/tar_split_linux_test.go

```diff
@@ -0,0 +1,104 @@
+package chunked
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"testing"
+	"time"
+
+	"github.com/containers/storage/pkg/chunked/internal"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/vbatts/tar-split/archive/tar"
+	"github.com/vbatts/tar-split/tar/asm"
+	"github.com/vbatts/tar-split/tar/storage"
+)
+
+func testTarheader(index int, typeFlag byte, size int64) tar.Header {
```

Comment:
The name made me thing this is itself a test, how about `createTestTarheader`?

Reply:
Thanks, fixed.